src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -151,7 +153,7 @@ static SimplexResult OPT_TryToCallSimplex(
             Probleme.Contexte = BRANCH_AND_BOUND_OU_CUT_NOEUD;
             Probleme.BaseDeDepartFournie = UTILISER_LA_BASE_DU_PROBLEME_SPX;
 
```

Comment:
I cant figure out, do we measure time or update it?

Reply:
We measure the time of the "update"